src/panel/resultTableStore.spec.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Row, RowGroup, RowItem } from './tableStore';
+import { observable } from 'mobx';
+import { _Region, filtersRow, filtersColumn } from '../shared';
+import { ResultTableStore } from './resultTableStore';
+import { log } from '../test/mockResultTableStore';
+import assert from 'assert';
+import { Result } from 'sarif';
+
+describe('ResultTableStore', () => {
+    const resultsSource = { results: log.runs![0].results! };
+    const selection = observable.box<Row | undefined>(undefined);
+    const filtersSource = {
+        keywords: '',
+        filtersRow: filtersRow,
+        filtersColumn: filtersColumn
+    };
+
+    it('should verify the default columns', () => {
```

Comment:
Your `describe`+`it` is effectively saying: "ResultTableStore should verify the default columns".

Reply:
attempted to fix